x-pack/plugins/ml/server/models/data_recognizer/modules/logs_ui_analysis/manifest.json

```diff
@@ -16,5 +16,9 @@
       "file": "datafeed_log_entry_rate.json",
       "job_id": "log-entry-rate"
     }
+  ],
+  "tags": [
```

Comment:
Updated in https://github.com/elastic/kibana/pull/168721/commits/ca79700393667389ca8e418f44990977db8817c3
I've added the `security` tag rather than removing all so this matches when `?filter=security` is used.